### ISSUE TITLE
strange match games of user wak017

### ISSUE THREAD

**Issue by rwbc** [glinscott/leela-chess#615]
current match ID 306

http://lczero.org/match_game/277318
http://lczero.org/match_game/277530
http://lczero.org/match_game/277731
and so on

**Comment by mooskagh** [comment]
It appears that wak017 has old version of a client (v0.8), and client version is not enforced for match games
(it's only enforced for training games).

Thanks for noticing!

**Comment by Wak017** [comment]
Whoa that's me.

I'll update my leela.

**Comment by ASilver** [comment]
Does that affect the ratings given to each network version?

**Comment by rwbc** [comment]
Please don't answer if you have no clue.
There is nothing to 'suppose' here, only training games were checked for right version until now.
That's why it is labelled as a bug and is explained above by mooskagh - why not just read the thread?

@Albert
The influence was negligible because only 7 faulty games were played with a score of 4:3.

**Comment by so-much-meta** [comment]
Here's another one that popped up today... http://lczero.org/match_game/297786

**Comment by so-much-meta** [comment]
And this game was scored as a draw, not sure why: http://www.lczero.org/match_game/298660

**Comment by ASilver** [comment]
It ended in a 3-fold repetition. Am I missing something?

**Comment by so-much-meta** [comment]
No, I immediately edited that comment to say it was a mistake (I was looking at multiple games at once, got confused) and then deleted it.
